phel-config.php

```diff
@@ -2,11 +2,16 @@
 
 declare(strict_types=1);
 
+use Phel\Config\PhelOutConfig;
+
 return (new \Phel\Config\PhelConfig())
     ->setSrcDirs(['src/phel'])
     ->setTestDirs(['tests/phel'])
     ->setVendorDir('vendor')
-    ->setOutDir('out')
+    ->setOut((new PhelOutConfig())
+        ->setDestDir('out')
+        ->setMainPhelNamespace('')
+        ->setMainPhpFilename('main'))
```

Comment:
No specific reason. I was considering index as well, but I choose main without any major argument. For me both are fine, but I will change it to index because your expectation.